test/specs/elements/Header/Header-test.js

```diff
@@ -69,7 +80,14 @@ describe('Header', () => {
   })
 
   describe('subheader', () => {
-    it('adds HeaderSubheader as child when there is an icon', () => {
+    it('adds HeaderSubheader as child', () => {
```

Comment:
This one should be covered by `implementsShorthandProp()` for `subheader`.  Should be able to remove it without a change in coverage.